### PR TITLE
detached tuner process under windows

### PR DESCRIPTION
**Pull request by J-shang**
Fix dispatcher exit before rest main process clean up. This issue due to CTRL BREAK or CTRL C is sent to all console processes that are attached to the console in Windows.

### CODE REVIEW

File: nni/experiment/experiment.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def run(self, port: int = 8080, debug: bool = False) -> bool:
             while True:
                 time.sleep(10)
                 status = self.get_status()
-                if status == 'STOPPED':
+                if status == 'DONE' or status == 'STOPPED':
```
Comment (SparkSnail) [inline nni/experiment/experiment.py:172]: Why add `DONE`? What is the behavior of `SUCCEED` or `FAILED`?

Comment (J-shang) [reply nni/experiment/experiment.py:172]: It is the status of experiment, not trials. And we expect this process automatic exit when the experiment is finished. You reminded me maybe we should also add `ERROR`?

Comment (SparkSnail) [reply nni/experiment/experiment.py:172]: Seems `ERROR` already exist in line 174.

**Review by SparkSnail — APPROVED** [review]

File: nni/experiment/experiment.py (hunk 1 above)
Comment (J-shang) [reply nni/experiment/experiment.py:172]: Oh yes, I miss it.

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**